dts/bindings/serial/nordic,nrf-uart-common.yaml

```diff
@@ -43,3 +43,27 @@ properties:
       description: |
         The CTS pin to use. The pin numbering scheme is the same as the
         tx-pin property's.
+
+    current-speed:
+      type: int
+      required: true
```

Comment:
Why is this being promoted to `required: true` from `required: false`?

Reply:
Currently, the driver does not allow changing the speed of an uart at runtime, so it is only possible to use the uart with this setting. The better solution would be to implement line control settings in the driver, so maybe this is a bad choice.

Reply:
OK, I see. Neither of the nRF UART drivers checks if the property is missing, so this is in fact required. I agree this is a good change, though it should probably also be in a commit of its own since it's fixing an unrelated issue to enumerating the possible values.

Reply:
> Currently, the driver does not allow changing the speed of an uart at runtime, so it is only possible to use the uart with this setting.

Which nRF UART driver does not allow that? There is no need to force current-speed property.